angular/src/lib/select/examples/select-default.component.ts

```diff
@@ -5,25 +5,43 @@ import { Component, OnInit } from '@angular/core';
   template: `
     <div class="medium-8 columns">
       <md-select
-        defaultValue="Select an option"
-        (select)="onSelect($event)"
-        buttonClass="my-custom-button-class"
-        [ngClass]="'angular-custom-class'"
-      >
-        <div md-select-option class="custom-class-prop" label="first option"></div>
-        <div md-select-option label="second option"></div>
+        [options]="people"
+        [(ngModel)]="person"
+        (handleChange)="onChange($event)"
+        optionLabel="name"
+        placeHolder="Select an option"
+        buttonClass="custom-button-class"
+        [buttonStyle]="{width: '80%'}"
+        overlayClass="testOverlayClass" >
       </md-select>
-  </div>
+    </div>
+
+  <p> ngModel: {{person ? person.name : 'none'}} <p>
   `,
   styles: []
 })
-export class ExampleSelectDefaultComponent implements OnInit {
+export class SelectDefaultComponent {
 
-  constructor() { }
+  person;
+  people;
 
-  ngOnInit() { }
+  constructor() {
+    this.people = [
```

Comment:
Was this an ask to move from the md-select-option to the array?  Just wondering since the input for feedback was the opposite. (i.e. they prefer to compose rather than inherit)

Reply:
wasn't an ask, i plan on making an example where they can pass in their own ng-template of whatever content they would like to show as a select option so its not just inherited